spec/features/users/04_user_sends_link_request_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+feature 'user send a story link request', %{
+  As an authenticated user
+  Request permission form another user to link stories
+  So new users can view my story link on that user's story show page
+} do
+
+  # Acceptance Criteria:
+  # [√] User sends a Link Request
+  # [√] Success notification displays
+
+  let!(:user) { FactoryGirl.create(:user) }
+  let!(:user2) { FactoryGirl.create(:user) }
+  let!(:story) { FactoryGirl.create(:story,
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.